index.html

```diff
@@ -0,0 +1,314 @@
+
+          <!-- Links -->
+          <a href="/index.html">
+            <h6 class="mr-6 cursor-pointer h6 text-rose-500">Home</h6>
+          </a>
+          <a href="/pages/product-listing-page/product-listing-page.html">
+            <h6 class="mr-6 cursor-pointer h6 text-white text-hover-rose-500">
+              Shop
+            </h6>
+          </a>
+          <!-- Links -->
+        </div>
+        <!-- header-left (logo, nav-links) -->
+
+        <!-- header middle (searchbar) -->
+        <form action="" class="search__bar__wrapper">
+          <input type="text" class="search__bar" placeholder="Search" />
+          <a href="#">
+            <i class="fas fa-search search__icon text-rose-500"></i>
+          </a>
+        </form>
+        <!-- header middle (searchbar) -->
+
+        <!-- header right (nav-links, login/signup CTA) -->
+        <div class="flex justify-center align-items-center">
+          <!-- login/signup CTA -->
+          <a
+            class="btn btn-square-solid btn-md text-white bg-rose-600 btn-bg-hover-rose-600 mr-7"
+          >
+            LOGIN
+          </a>
+          <!-- login/signup CTA -->
+
+          <!-- navigation links -->
+          <a href="/pages/wishlist-page/wishlist-page.html">
+            <span class="w-full h-full tooltip btn-icon badge-container mr-3">
+              <i
+                class="text-rose-500 text-hover-rose-300 fas fa-heart text-3xl"
+              ></i>
+              <span
+                class="icon-badge-sm bg-slate-100 text-black text-xs status-badge"
+                >1</span
+              >
+              <span
+                class="tooltip-text text-black text-lg tooltip-text-bottom bg-rose-100"
+              >
+                Wish List
+              </span>
+            </span>
+          </a>
+          <a href="pages/cart-page/cart-page.html">
+            <span class="btn-icon badge-container tooltip"
+              ><i
+                class="text-rose-500 text-hover-rose-300 fas fa-shopping-cart text-3xl"
+              ></i>
+              <span
+                class="icon-badge-sm bg-rose-500 text-white text-xs status-badge bg-slate-100 text-black"
+                >2</span
+              >
+              <span
+                class="tooltip-text text-black text-lg tooltip-text-bottom bg-rose-100"
+              >
+                Cart
+              </span>
+            </span>
+          </a>
+          <!-- navigation links -->
+        </div>
+        <!-- header right (nav-links, login/signup CTA) -->
+      </div>
+    </header>
+    <!-- header -->
+
+    <!-- image slider -->
+    <section class="p-20 relative" style="top: 7rem">
+      <div class="slideshow rounded-lg overflow-hidden" style="height: 30rem">
+        <div class="slides fade">
+          <div class="slide-number">1 / 3</div>
+          <img
+            src="https://picsum.photos/id/1015/500/200"
+            style="width: 100%; object-fit: contain"
```

Comment:
Please try to avoid inline styling.

Reply:
I have added inline styles so the classes from my component lib are overwritten , i wanted a few properties to overwrite the styles being applied from my component lib. As inline styles are more specific i have used them here